.circleci/config.yml

```diff
@@ -603,6 +607,27 @@ jobs:
           file: ./api-unit-tests.xml
           flags: api
 
+  add-to-metrics:
+    docker:
+      - image: cimg/base:2020.10
+    resource_class: small
+    steps:
+      - run:
+          name: PUT release time to OPG metrics
+          command: |
+            EPOCH_MILLISECONDS=$(date +%s%3N)
+
+            curl --location --request PUT ${METRICS_ENDPOINT} \
```

Comment:
Worth checking the default curl timeout on circle if using this method.

Reply:
I can't see anything curl specific but the default timeout for a command on CircleCI is 10 minutes. Have you run into this issue before @mattmachell?

Reply:
Was just thinking in terms of builds failing quickly and feedback cycles, and also how we handle failure modes here. I'd set in at 30 seconds to avoid builds hanging. `-m 30` for 30 seconds timeout (https://ec.haxx.se/usingcurl/usingcurl-timeouts).

Reply:
Ah I was coming at it from the opposite perspective 😂 makes sense, I'll add that in